src/types.ts

```diff
@@ -88,4 +88,10 @@ declare global {
     url: string;
     element?: Element;
   }
+
+  // https://w3c.github.io/longtasks/#performancelonganimationframetiming
```

Comment:
This has just moved.

```suggestion
  // https://w3c.github.io/long-animation-frames/#performancelonganimationframetiming
```